natchez-extras-testkit/src/main/scala/com/ovoenergy/natchez/extras/testkit/TestEntryPoint.scala

```diff
@@ -15,42 +15,50 @@ import java.time.Instant
  * and additionally lets you see all the completed spans
  */
 trait TestEntryPoint[F[_]] extends EntryPoint[F] {
-  def spans: F[List[TestSpan]]
+  def spans: F[List[CompletedSpan]]
 }
 
 object TestEntryPoint {
 
-  case class TestSpan(
-    exitCase: ExitCase,
+  case class CompletedSpan(
+    tags: List[(String, TraceValue)],
     parent: Option[String],
```

Comment:
Would it be great to preserve the tree structure and store `CompletedSpan` as a parent itself ?